### PR TITLE
4429

### PR DESCRIPTION
**Pull request by ethankaplan**
## MADiE PR

Jira Ticket: [MAT-4429](https://jira.cms.gov/browse/MAT-4429)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/measureGroups/MeasureGroups.tsx [hunk 1]
```diff
@@ -13,6 +13,8 @@ import MeasureGroupPopulationSelect from "./MeasureGroupPopulationSelect";
 import * as _ from "lodash";
 import { MeasureGroupSchemaValidator } from "../../validations/MeasureGroupSchemaValidator";
 import { useOktaTokens } from "@madie/madie-util";
+import { optionGroupUnstyledClasses } from "@mui/base";
+import { groupBy } from "lodash";
```
Comment (RohitKandimalla) [inline src/components/measureGroups/MeasureGroups.tsx]: I couldn't find usage of these imports, Are we required to have them ?

File: src/components/measureGroups/MeasureGroups.tsx [hunk 2]
```diff
@@ -106,6 +130,22 @@ export const DefaultPopulationSelectorDefinitions = [
   },
 ];
 
+export const MeasureImprovmentNotation = [
```
Comment (RohitKandimalla) [inline src/components/measureGroups/MeasureGroups.tsx]: Typo _MeasureImprovementNotation_

Comment (RohitKandimalla) [reply src/components/measureGroups/MeasureGroups.tsx]: Any reason, why we may need to export it?

File: src/components/measureGroups/MeasureGroups.tsx [hunk 3]
```diff
@@ -484,19 +559,97 @@ const MeasureGroups = () => {
...
+                        autoComplete="rate-aggregation"
+                        placeholder="Rate Aggregation"
+                        data-testid="rateAggregationText"
+                        {...formik.getFieldProps("rateAggregation")}
+                      />
+                    )}
+                  </FieldSeparator>
+                  <Divider />
+                  <FieldLabel htmlFor="rate-aggregation">
+                    Improvment Notation
```
Comment (RohitKandimalla) [inline src/components/measureGroups/MeasureGroups.tsx]:
```suggestion
                    Improvement Notation
```

File: src/components/measureGroups/MeasureGroups.tsx [hunk 4]
```diff
@@ -484,19 +559,97 @@ const MeasureGroups = () => {
...
+                              key={`${opt.code}-${i}`}
+                              value={opt.label}
+                              data-testid="improvement-notation-option"
+                            >
+                              {opt.label}
+                            </option>
+                          )
+                        )}
+
+                        {/*!canEdit && formik.values.improvementNotation*/}
```
Comment (RohitKandimalla) [inline src/components/measureGroups/MeasureGroups.tsx]: Is this required for any future use?

**Review by RohitKandimalla — CHANGES_REQUESTED** [review]

File: src/components/measureGroups/MeasureGroups.tsx (hunk 2 above)
Comment (ethankaplan) [reply src/components/measureGroups/MeasureGroups.tsx]: I copied syntax for the DefaultPopulationSelectorDefinitions above. Uncertain if export is needed

File: src/components/measureGroups/MeasureGroups.tsx (hunk 4 above)
Comment (ethankaplan) [reply src/components/measureGroups/MeasureGroups.tsx]: Removed

File: src/components/measureGroups/MeasureGroups.tsx (hunk 1 above)
Comment (ethankaplan) [reply src/components/measureGroups/MeasureGroups.tsx]: Removed

File: src/components/measureGroups/MeasureGroups.tsx (hunk 2 above)
Comment (RohitKandimalla) [reply src/components/measureGroups/MeasureGroups.tsx]: aah ok, no problem. I just wondered if there is any other instance that is using it.

**Review by gregory-akins — APPROVED** [review]

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**